### PR TITLE
Fix puffin feature listing

### PR DESCRIPTION
**Pull request by repi**
Was missing a featured image and description, failing CI

### CODE REVIEW

**Comment by repi** [comment]
Yay works, Netlify preview links in the CI are so nice.

image

File: static/data/data.json [hunk 1]
```diff
@@ -150,7 +150,9 @@
       "emoji": "🐦",
       "tags": ["rust", "tools"],
       "description": "Simple instrumentation profiler for Rust.",
-      "featured": true      
+      "featured": true,
+      "extendedDescription": "The friendly little instrumentation-based CPU profiler for Rust",
+      "featureImage": "https://github.com/EmbarkStudios/puffin/blob/main/puffin.jpg?raw=true        "
```
Comment (vojd) [inline static/data/data.json:155]: Those spaces at the end of the `featureImage` URL ;)

Comment (repi) [reply static/data/data.json:155]: oh weird, does still work though but will fix

Comment (repi) [reply static/data/data.json]: there

**Review by celialewis3 — APPROVED** [review]

**Pull request merged**